docs/run/sessions.mdx

```diff
@@ -9,7 +9,7 @@ description: An overview of sessions and when to use them.
 A session is a Qiskit Runtime feature that lets you efficiently run multi-job iterative workloads on quantum computers. Using sessions helps avoid delays caused by queuing each job separately, which can be particularly useful for iterative tasks that require frequent communication between classical and quantum resources.
 
 <Admonition type="note">
-The queuing time does not decrease for the first job submitted within a session. Therefore, a session does not provide any benefits when running a single job.
+The queuing time does not decrease for the first job submitted within a session. Therefore, a session does not provide any benefits when running a single job. Additionally, sessions do not work on simulators because they do not have a queue.
```

Comment:
```suggestion
The queuing time does not decrease for the first job submitted within a session. Therefore, a session does not provide any benefits when running a single job. Additionally, sessions do not work on simulators because simulators do not have a queue.
```

Reply:
small clarification